module_7/demoblaze_tests/common/utils.py

```diff
@@ -0,0 +1,8 @@
+import time
+
+def generate_credentials() -> tuple[str, str]:
```

Comment:
This is not a error, the option use credentials as tuple is quite workable, but I would use `dataclass` for this purpose. See https://docs.python.org/3/library/dataclasses.html

```python
from dataclasses import dataclass

@dataclass
class Credential:
    username: str
    password: str
```